src/main/java/org/wirvsvirus/locoronando/category/Category.java

```diff
@@ -0,0 +1,42 @@
+package org.wirvsvirus.locoronando.category;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Category {
+  @Id
+  private UUID uniqueId;
+  private String name;
+  private String description;
+  private int handlerId;
```

Comment:
What is the handlerId?

Reply:
The handlerId is in this case the dealerId to filter them

Reply:
Don't get it. If it's referencing the dealer, shouldn't we then just reference the Dealer?